README.md

```diff
@@ -59,33 +59,43 @@ Table of Contents
 
 Created by [gh-md-toc](https://github.com/ekalinin/github-markdown-toc)
 
-## Setup:
-The following `$SHELL` variables should be defined:
+## Setup
 
-- `$BUILD_HOME`
-- `$DATA_PATH`
-- `$ELOG_PATH`
+The `$SHELL` variable `$ELOG_PATH` should be defined:
+
+```
+export ELOG_PATH=/your/favorite/elog/path
+```
 
 Then execute:
 
 ```
-source $BUILD_HOME/cmsgemos/setup/paths.sh
-source $BUILD_HOME/gem-plotting-tools/setup/paths.sh
+source gem-plotting-tools/setup.sh -c 0.3.1 -g 1.0.0 -G 5 -v 2.0.0 -V 3
```

Comment:
I am not sure this script should not belong to this repository.  Ideally it should belong to a future, not yet created, meta repo which will handle the job of checking cross-repo compatibility at both SW & FW level (e.g. Repo A version X.Y.Z is compatible with repo B version A.B.C and expects FW version 1.2.3).  I am not sure it make sense to have a tool the user will use to get the code inside the code; especially one that takes the other repos.

Until this repo is created by @jsturdy and @mexanick I think we need another strategy.  My view would be to create the above mentioned repo and have this script be the first commit.  Then it could be expanded upon from there.  The instructions would then still reference this script but *not* in the repo.

Perhaps @jsturdy or @mexanick could comment on this issue.

Reply:
It should not, especially as the only dependency from `gemplotting` is on the `utils` of `cmsgemos_gempython` (i.e., *no* dependence on `vfatqc`)

Furthermore, for a pure `USER`, no checking out of a `git` repository should ever need to be done.

To this end, I'm also prototyping setting up a `PyPi` repository that will have the packages pushed to, either on the official site, or setting up a private one as part of the CERN hosted `PaaS` ecosystem being developed for our uses

Reply:
Okay for the short-term I would suggest to follow @jsturdy's statement.  To that end for the short-term:

1. remove `setup.sh` from the repo,
2. restore the previous `*.sh` scripts,
3. update the `README.md` to download the script at the gist and have the user follow those,
4. do not close related setup/documentation issues as we will return to this later.

This will allow the continued analysis of the channel loss study on p5 data and will keep open the further developments.